_functions/synth_parameters/legato.md

```diff
@@ -0,0 +1,16 @@
+---
+title: legato
+category: synth_parameters
+---
+
+Controls the length of the sound relative to its "space" in the pattern.  `legato "1"` means the sound will play for the
```

Comment:
maybe it would be good to mention the technical term "inter onset time": the time from the beginning of one sound to the beginning of the next.